README.md

```diff
@@ -50,7 +50,7 @@ Kubernetes Fury Monitoring provides the following packages:
 | [prometheus-operated](katalog/prometheus-operated)     | `2.36.1` | Prometheus instance deployed with Prometheus Operator's CRD                                                 |
 | [alertmanager-operated](katalog/alertmanager-operated) | `0.24.0` | Alertmanager instance deployed with Prometheus Operator's CRD                                               |
 | [blackbox-exporter](katalog/blackbox-exporter)         | `0.24.0` | Prometheus exporter that allows blackbox probing of endpoints over HTTP, HTTPS, DNS, TCP, ICMP and gRPC.    |
-| [grafana](katalog/grafana)                             | `8.5.5`  | Grafana deployment to query and visualize metrics collected by Prometheus                                   |
+| [grafana](katalog/grafana)                             | `9.3.6`  | Grafana deployment to query and visualize metrics collected by Prometheus                                   |
```

Comment:
Let's leave this info on the main README as the previous one for now, we will update the main README when we create the release note for the new version

Reply:
IMHO is OK to update the readme as part of this PR. we usually do it all at once, but now that we are updating piece by piece we can update the readme with each PR. So the document reflects the status of the branch.